changelog/63402.fixed

```diff
@@ -0,0 +1 @@
+Repaired zypper repositories being reconfigured without changes
```

Comment:
This was just a recent change, but the file extension needs to be changelog/63402.fixed.md

I can make this change if you want.

Reply:
Done!